spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,9 +31,16 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:wixtestfiles) { "/tmp/spec/fixtures/wix/resources/windows/wix" }
 
       before do
         cur_plat.instance_eval(plat[:block])
+        FileUtils.mkdir_p(wixtestfiles)
+        FileUtils.cp_r("spec/fixtures/wix/resources/windows/wix", "#{wixtestfiles}/..")
+      end
+      after do
+        FileUtils.rm_rf("#{WORK_BASE}")
+        FileUtils.rm_rf("/tmp/spec")
       end
```

Comment:
@jcoconnor I was wondering why these aren't a part of lines 63-76, where we are already doing a lot of set up for the wix tests. We don't need this to happen for any of the non-wix tests, which it currently is. It's not a question of `:all` vs. `:each`